.travis.yml

```
@@ -3,10 +3,16 @@ language: java
 env:
   global:
     - VERIFY_USE_PUBLIC_BINARIES=true
+    - secure: "uNn96syCzMmn/AkH/CUmHHABmQdLHdKje1pFM2+VR+1dgrd1UavKe+ICMCX1uJGZdcpuDUzkhYB46IVwIA1YalG7ZLFr0WdkLeK+mwdFb9LnpOTWFhOvvNYTQXCplDJl07AzP8aqgCqx3q2oHtKCKGA35G+5EK1dI86rQr/Wp27cKTfojY08KXHRImtYqFrYaUvSxfRDyJFfPipJA7ktNkE8uZVYU4TLY1v6VGkcLfme+NI4dSMI5CV8Nr2aYavdowC/HedfQg8tO/zPEV+TJqdXuXNac23xmCI2I2QUM6T1dZr1QFTaGZ3tB1PVx4aUKeli0uOARtBmmZWuGfHEG/T9PaG7Zv+cjQJ0FdqE7BwvwBLWFGDlIU+fzuIH5+O4Qz0fobQBp9SMZW6uXPNon1rXg5qMypLeI5I6yqQql1UpExhwtytbicIs1k1+k4K9T+ADNcJ6e6kTU0pGu8PmnETkBm5CPgMnHmKmObeaNk2K0tw25Kbi5fmb3xaNi6iYxFz3QjBDxcZTy3OkH7jAMOlNylzokpqCDhV1adBZisVzOZ5eA/CZ1aUHizVL5wzj27jQ4WCiL/X0zHHDYr+JT1Huv3G4azZOVq7Hfc4Hd93BCg9ixC6ZYZ6NOI5Ue7DEESwKctFKJLQ2W3/h2yyOw068mASqXlA8TRbnEP8Q1Oo="
+    - secure: "E/Ui52SP8jC9kvP9RxYcvrP2Q3BPyKdi/almVWEfWetBKDH7g9M1bl6pgTLTrI31CZGS0O2MtOC05vTp7KkksveBwhbvXLOcb934bLpeOP0son3gXDH5ANTwOq98amFlyKbqzZre0BUMeIPtHc1ZoGc5s6q5a+nZ78zgyjiDVc9StlRx38QcF0rM7S9zzcIqu0eNoLTI4pDI/cP0STpIG68rsGKQSVUI5dFAGC0ZdVIEH7rSP/G5rPbX4scvwiQ2iBaPuaOkPvgfAW7Th4reQK3/97C+ShfC3aNfPfCYZoSYBROHSrgFqvSsEOGjiKnuqaz07RcGsh1tAXLrzcYVey3YExxiPVIYEY8q5P5nCjE0pT65AJqnnXT0gJ/+zLV9I5x4DPEBRcgURDAhVgMRTlOobE0VLb2rvsTdUZpiopVoAQUQEH1s8rD1LA1EqethRI3IHBv9jK63ca0gwwQcKh6KxZE8YxjsiVTXPjRu64XFz0OlH4PBn7ZNja/b0aGeaQt/qoxGHTQ3/xkuiVArg1WfVKo1ZW9jOG5R2vOiV6Lbxkmcb+pkMUF9X8JvzUYNUjNFqHA2119lXbamVKG2gAzi6dlmFldpyyU8rdJONdrRmdAlAKq10Cre2QEPFzh1ehHIMrRVlxT0pi7BQM7aH2sKZzFeifhpL5+0J698emo="
```

```suggestion
    - secure: "RWxtPrNRQkNZB4pW6FTXti9WdNntvaevSvk6V/E37XLoXJlqn7GxPcNSU2WBKY0kC7nq0QEkR44vPJwQ5zJvr5LFpwRjI5zw6WVeFV8N2n7zNDxIAD4zfUzJOzWmEK6ZJhtL5ISqjf/9L++VcbOys4pJX/2IIeqIVlivNCaOou5cj1owPaG80otzfcpH+aX2voSxpAtc2pyzWfN30O64NCJIdmo1fuaoQuNK2lqp+5NQhCqWDgWVnrGmw06Brwcw4XodvMGzNqbASq+o2mc8+vFQHI9wLScrRA4X3JfvHho5icUQoeKN7/ohRzBs+BoU/uURUfKfIam6cObJbFlcn9UlXS9l9LbpRaiUxvnRaXoas5OE3mSnCD+UfgzFf7cFYBSLcsQNOO5pzxvGGc8NzMN3XVo13vDAf6SOCfiR4cjKx9YdXvK10aHOA72zY9giFcmWieocT8Yth+EbXi7jshSuYKEB5ig8tqH4Os4vXVKNGqow4vxZf4f9AxeAK8mBAlD5Mbd00JU2euzkOF3DSgzEb1uxg4BzM7YBrm74b7F3X2GO0mDsKsP8ZsLy0p+cLORXSrcPt3wQTkDMo7jGaVkalxKCmTUU48q4v/5VLGMMVTbU861/5flC/CLGU19ABFnCG46AorcDagVcsTA9EOqllHVi2KwJPVxLr+CFx88="
```

Updated GITHUB_PAT signed for travis-ci.com